### ISSUE TITLE
When manually hidden, the KI will not stay hidden if there is an incoming chat message.

### ISSUE THREAD

**Issue by DamnBriggsy** [H-uru/Plasma#1389]
On the vanilla client, clicking the button to hide the KI (pictured) will keep the KI hidden until it is re-opened by the user. On the H'uru client, the KI will re-appear any time a new chat message comes in.

![image](https://user-images.githubusercontent.com/70078061/235688730-e82ede12-7f0a-4e95-bda9-242b53a55845.png)

I propose the functionality be changed to match the vanilla client: choosing to hide the KI should keep it hidden until the user chooses to show it again. This would be of particular use when taking video or screenshots in situations where there is a lot of chatter and the KI being visible is not desired.

**Comment by dpogue** [comment]
Agreed, with the note that if a chat message with a name mention comes in while the KI is minimised, we should flash the KI icon.

**Comment by TheScarFr** [comment]
Agreed with both.

On the same subject, I propose this:
If my KI is visible but not opened (I can read the chat), and I link to another Age, I'd like my KI to still be visible when I appear in the other Age.
The book/stone GUI minimizes the KI, but in a previous version of MOUL (or in offline Uru, I don't remember) the KI did show again after the link if it was visible before the link, which is what I'd like.

**Comment by patmauro** [comment]
Piggybacking off of this to add that the non-KI, beginners "chat" interface also needs a similar feature! Currently (in both clients) when a player starts in the Cleft with default settings, the subtitles from the Zandi tutorial will immediately burn into their screen and will stay on screen forever, they do not fade out and will not disappear unless the player types /clearchat (a function they, as a newcomer, probably aren't aware of). This is because there is no way to hide this chat window like you can do with the KI. A good solution to this to make the UI a little more player-friendly would be to add the ability to hide the "vanilla" chat like you can do with the KI. (Perhaps just with a small "X" graphic.)

Had been considering adding this as a separate ticket but this seems worth mentioning here.

**Comment by TheScarFr** [comment]
Speaking about the /[something] commands, it could be useful to add a button in the "Options Menu" GUI, to open a popup that lists all the /[something] commands and their purpose.
